packages/core/src/components/button/_common.scss

```diff
@@ -184,6 +188,10 @@ $non-default-state-selectors: ":last-child, :hover, :active, .pt-active, [class*
     background-image: none;
     color: $dark-button-color-disabled;
   }
+
+  .pt-button-spinner-container .pt-spinner-head {
```

Comment:
i suspect `.pt-button-spinner-container` is unnecessary in this selector and can be safely removed. 

will there ever be another `.pt-spinner-head` inside a button? if so, would you ever want that head to be a _different_ color?